go/base/static/templates/apps/dialogue/states/choice/edit.jst

```diff
@@ -1,5 +1,6 @@
 <div class="form-group">
   <label class="control-label">Define the question</label>
+  <textarea class="text form-control"><%= model.get('text') %></textarea>
```

Comment:
Could we do this for the end and freetext states too (and remove their `TextEditView` partial)?